assets/js/modules/analytics-4/datastore/partial-data.test.js

```diff
@@ -237,7 +237,7 @@ describe( 'modules/analytics-4 partial data', () => {
 				).toEqual( 20201220 );
 			} );
 
-			it( 'uses a resolver to determine the date and if the date is available. save it to server', async () => {
+			it( 'uses a resolver to determine the date and if the date is available. saves it to server', async () => {
```

Comment:
I think we'd want to use a comma here instead of a period.

```suggestion
			it( 'uses a resolver to determine the date and if the date is available, saves it to server', async () => {
```